assets/scss/_styles_project.scss

```diff
@@ -1404,6 +1404,10 @@ ul.sectionlist > li:hover {
   margin: 2rem auto;
 }
 
+#pagination2 {
+  margin: 2rem auto;
+}
```

Comment:
I think there's already a #pagination item somewhere in this file, can you merge the two like we did for mr-component and mr-model?